cmsplugin_filer_image/static/cmsplugin_filer_image/css/filer_image.css

```diff
@@ -1,6 +1,28 @@
 .filer_image_container{
     margin-bottom: 12px;
     position: relative;
+    display: block;
+    max-width: 100%;
+}
+
+.filer_image_container .image-link {
+    display: block;
+    max-width: 100%;
+}
+
+.filer_image_container.has-credit,
+.filer_image_container.has-caption {
```

Comment:
This will affect the elements around the image plugin, not the caption/credit elements positioning since they are contained by .filer_image_container.has-caption element